Concentration/Concentration.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Concentration.swift
+//  Concentration
+//
+//  Created by Oren Dinur on 08/02/2022.
+//
+
+import Foundation
+
+class Concentration{
```

Comment:
Please follow this style:
```suggestion
class Concentration {
```
Make changes in other places in your code if needed.